backend/database-router.test.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-env mocha */
+
+import assert from 'node:assert'
+import RouterBackend from './database-router.js'
+import { cloneDeep, omit } from 'turtledash'
+
+const validConfig = {
```

Comment:
The config is valid, but my general comment about this is that the config isn't tested as far as I can see. It'd be useful to perhaps add a test verifying that keys go to the correct storage.

Reply:
So I've added various tests for `lookupBackend()`, which should implicitly test if keys go to the correct storage, while delegating the actual `read-`, `write-` and `deleteData()` tests to underlying backend tests (which currently go in `database.test.js`)